contributors/devel/architectural-roadmap.md

```diff
@@ -913,7 +913,7 @@ discovering the following:
 * Initializers and finalizers
 
 * [Scheduler
-  extensions](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduler_extender.md)
+  extensions](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/scheduler_extender.md)
```

Comment:
same repo. This can be:
`extensions](/contributors/design-proposals/scheduling/scheduler_extender.md)`